examples/lifetime_prediction/lifetime_feature_generator.py

```diff
@@ -17,8 +17,12 @@ def __init__(self, h5py_hdf_path, data_csv_path):
         self.data_csv_path = data_csv_path
 
     @will_generate('memory', 'data_df')
-    def gen_data_df(self):
-        return {'data_df': pd.read_csv(self.data_csv_path, index_col='id')}
+    def gen_data_df(self, args=None):
```

Comment:
should add a @params('nrows') here, and we don't support default value now, so you should give it a value.

Reply:
next step, I will limit the args to only the variables declared in @params, and the default value is maybe the next next ... next step

Reply:
my "default value" here means you should never use `args.get()`. `args=None` is definitely wrong. You will always get this value in the future.

Reply:
You don't understand...

Reply:
You should never use args.get(). args=None is definitely wrong. You will always get this value in the future.
You are still using them.

Reply:
and you didn't add `@params`